src/main/java/org/taktik/icure/entities/Contact.java

```diff
@@ -173,7 +174,15 @@ public void setGroupId(String groupId) {
 
 	public void setExternalId(String externalId) {
 		this.externalId = externalId;
-	}
+    }
```

Comment:
Something strange with formatting... Looks like 4 spaces used by default here.

Reply:
I'm using VSCode with detect indentation & fallback to 4 spaces if no other config is available. I don't see any differences in the editor when opening though. 

I opened with Emacs and it does detect bat indenting in more places. I changed this.